frontend/src/metabase/entities/containers/rtk-query/EntityListLoader.tsx

```diff
@@ -66,7 +66,7 @@ interface Props<Entity, EntityWrapper> {
   loadingAndErrorWrapper?: boolean;
   LoadingAndErrorWrapper?: ComponentType<LoadingAndErrorWrapperProps>;
   pageSize?: number;
-  reload?: boolean;
+  reload?: boolean | (() => void); // reload can be passed as a callback from the outer loader
```

Comment:
:man_facepalming:  :+1:

Reply:
```suggestion
  reload?: boolean | ChildrenProps["reload"]; // reload can be passed as a callback from the outer loader
```

The same for `EntityObjectLoader`

Reply:
Fixed